<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabstoragestockwindow</class>
 <widget class="QWidget" name="tabstoragestockwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>库存信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tabstock_tablecombox">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>位 置</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>管 料</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tabstock_choicecombox">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>位置编号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>管 料</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>数 量</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>状 态</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其 他</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tabstock_optioncombox">
       <item>
        <property name="text">
         <string>模糊查询</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>精确查询</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>查询输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tabstock_choicetext"/>
     </item>
     <item>
      <widget class="QPushButton" name="tabstock_querybutton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>查 找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tabstock_submitbutton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>提交修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tabstock_deletebutton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>删除选中</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tabstock_cancelbutton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>撤销修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="tabstock_table">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tabstock_lastbutton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>上一页</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tabstock_currenttext">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tabstock_totaltext">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tabstock_nextbutton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>下一页</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>共</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tabstock_totalnumtext">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>条记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
